From 95f7e0acce4aa778e6c2c3103f36c8aa8142fc70 Mon Sep 17 00:00:00 2001
From: Anca-Jeanina Floarea <anca.floarea@freescale.com>
Date: Mon, 18 Feb 2013 21:50:00 +0000
Subject: [PATCH 470/518] dpa_ipsec: Remove unused variables

The global variables "pcd_df_action",
the macro "PCD_DF_ACTION" and the enum "dpa_ipsec_df_action"
are no longer used in the DPA IPSec component and can be removed.

Signed-off-by: Anca Jeanina FLOAREA <anca.floarea@freescale.com>
[Grabbed from the branch, LINUX_IR5.2.0, of
https://git.freescale.com/git-private/cgit.cgi/ppc/alu-b4860/linux.git.]
Signed-off-by: Tiejun Chen <tiejun.chen@windriver.com>
---
 drivers/staging/fsl_dpa_offload/dpa_ipsec.c |    3 ---
 drivers/staging/fsl_dpa_offload/dpa_ipsec.h |   11 -----------
 include/linux/fsl_dpa_ipsec.h               |    8 --------
 3 files changed, 0 insertions(+), 22 deletions(-)

diff --git a/drivers/staging/fsl_dpa_offload/dpa_ipsec.c b/drivers/staging/fsl_dpa_offload/dpa_ipsec.c
index 47fa2a3..3a4d1ea 100644
--- a/drivers/staging/fsl_dpa_offload/dpa_ipsec.c
+++ b/drivers/staging/fsl_dpa_offload/dpa_ipsec.c
@@ -40,9 +40,6 @@
 #include "fm_common.h"
 #include "fm_pcd.h"
 
-/* DPA IPsec Mapping between API DF bit action and FM PCD DF bit action */
-e_FmPcdManipDontFragAction pcd_df_action[] = PCD_DF_ACTION;
-
 /* DPA IPsec Mapping between API algorithm suites and SEC algorithm IDs */
 struct ipsec_alg_suite ipsec_algs[] = IPSEC_ALGS;
 
diff --git a/drivers/staging/fsl_dpa_offload/dpa_ipsec.h b/drivers/staging/fsl_dpa_offload/dpa_ipsec.h
index 66309d7..5d345e4 100644
--- a/drivers/staging/fsl_dpa_offload/dpa_ipsec.h
+++ b/drivers/staging/fsl_dpa_offload/dpa_ipsec.h
@@ -52,17 +52,6 @@
 #include "fm_pcd_ext.h"
 #include "cq.h"
 
-
-/* MACRO declaration */
-#define PCD_DF_ACTION	{				\
-	/* DPA_IPSEC_DF_ACTION_DISCARD */		\
-	DPA_CLS_HM_DF_ACTION_DROP,			\
-	/* DPA_IPSEC_DF_ACTION_OVERRIDE */		\
-	DPA_CLS_HM_DF_ACTION_FRAG_ANYWAY,		\
-	/* DPA_IPSEC_DF_ACTION_CONTINUE */		\
-	DPA_CLS_HM_DF_ACTION_DONT_FRAG			\
-}
-
 #define OP_PCL_IPSEC_INVALID_ALG_ID	0xFFFF
 
 #define IPSEC_ALGS_ENTRY(enc, auth)	{		\
diff --git a/include/linux/fsl_dpa_ipsec.h b/include/linux/fsl_dpa_ipsec.h
index 2be976d..f3854bf 100644
--- a/include/linux/fsl_dpa_ipsec.h
+++ b/include/linux/fsl_dpa_ipsec.h
@@ -400,14 +400,6 @@ int dpa_ipsec_remove_sa(int sa_id);
  */
 int dpa_ipsec_flush_all_sa(int dpa_ipsec_id);
 
-/* DPA-IPSec Action For DF Bit */
-enum dpa_ipsec_df_action {
-	DPA_IPSEC_DF_ACTION_DISCARD = 0,	/* Discard the frame	      */
-	DPA_IPSEC_DF_ACTION_OVERRIDE,		/* Override DF bit	      */
-	DPA_IPSEC_DF_ACTION_CONTINUE		/* Continue processing without
-						 * performing IP Fragmentation*/
-};
-
 struct dpa_ipsec_l4_params {
 	uint16_t src_port;	/* Source port				      */
 	uint16_t src_port_mask;	/* Source port mask			      */
-- 
1.7.5.4

